Mfalm3/Questioner
# #163047438 directory structure

#### What does this PR do?

This pull request creates the recommended app structure

#### Description of tasks to be completed

The following are the tasks to be completed  
Create the recommended app structure

#### How should you manually test this?

Clone this repository  
`$ git clone https://github.com/Mfalm3/Questioner.git`

Cd into the directory  
`$ cd Questioner`

Change into the current branch  
`$ git checkout ch1-directory-structure-163047438`
check the directory structure

Pivotal tracker story associated with this PR:  
[#163047438](https://www.pivotaltracker.com/n/projects/2235279)

![image](https://user-images.githubusercontent.com/18320823/50853106-66a8c400-1392-11e9-81f1-58cafbd93029.png)
